### PR TITLE
[docs]: Tweaks, capitalization and fixing typos

### PR DESCRIPTION
**Pull request by Caladan08**
Tiny tweaks to sentences, fixed typos, capitalised few words and improved sentence structure

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing guide](https://github.com/svelteuidev/svelteui/blob/main/CONTRIBUTING.md)
- [x] Prefix your PR title with `[@svelteui/core]`, `[@svelteui/actions]`, `[@svelteui/motion]`, `[@svelteui/core]`, `[core]`, or `[docs]`.
- [x] This message body should clearly illustrate what problems it solves.

### Tests

- [ ] Run the tests with `npm test` and lint the project with `yarn lint` or just run `yarn repo:prepush` and check to see if it's passing.

### CODE REVIEW

File: apps/docs/src/pages/core/image.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ docs: 'core/image.md'
 
 ## Usage
 
-Image component is a wrapper around img element with option to change object fit, radius and placeholder:
+Image component is a wrapper around an `img` element with the option to change object fit, radius and placeholder:
```
Comment (BeeMargarida) [inline apps/docs/src/pages/core/image.md:22]: Just a small fix:
```suggestion
Image component is a wrapper around an `img` element with the option to change object-fit, radius and placeholder:
```

File: apps/docs/src/pages/core/overlay.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ docs: 'core/overlay.md'
 ## Usage
 
 Overlay had absolute position and will take 100% of width and height of parent container.
-It is used to build components like [Modal (not yet implemented)](/core/modal/) and [LoadingOverlay (not yet implemented)](/core/loading-overlay/).
+It is used to build components like [Modal](/core/modal/) and [LoadingOverlay (not yet implemented)](/core/loading-overlay/).
```
Comment (BeeMargarida) [inline apps/docs/src/pages/core/overlay.md:23]:
```suggestion
It is used to build components like [Modal](core/modal/) and [LoadingOverlay (not yet implemented)](core/loading-overlay/).
```

**Review by BeeMargarida — COMMENTED** [review]
Thank you for these fixes! 🙏

**Comment by Caladan08** [comment]
Done!

**Review by BeeMargarida — APPROVED** [review]

**Pull request merged**